Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/BearerTokenIdentityResolvers/SSOBearerTokenIdentityResolver.swift

```diff
@@ -0,0 +1,94 @@
+//
```

Comment:
This token identity resolver fetches the access token from cached SSO token file.